axelor-crm/src/main/java/com/axelor/apps/crm/message/MessageServiceCrmImpl.java

```diff
@@ -59,38 +63,33 @@ public Message createMessage(Event event)
 
     // Get template depending on event type
     Template template = null;
+    Company company = event.getUser().getActiveCompany();
 
-    switch (event.getTypeSelect()) {
-      case EventRepository.TYPE_EVENT:
-        template =
-            Beans.get(CrmConfigService.class)
-                .getCrmConfig(event.getUser().getActiveCompany())
-                .getEventTemplate();
-        break;
+    if (company != null) {
+      switch (event.getTypeSelect()) {
+        case EventRepository.TYPE_EVENT:
+          template = Beans.get(CrmConfigService.class).getCrmConfig(company).getEventTemplate();
+          break;
 
-      case EventRepository.TYPE_CALL:
-        template =
-            Beans.get(CrmConfigService.class)
-                .getCrmConfig(event.getUser().getActiveCompany())
-                .getCallTemplate();
-        break;
+        case EventRepository.TYPE_CALL:
+          template = Beans.get(CrmConfigService.class).getCrmConfig(company).getCallTemplate();
+          break;
 
-      case EventRepository.TYPE_MEETING:
-        template =
-            Beans.get(CrmConfigService.class)
-                .getCrmConfig(event.getUser().getActiveCompany())
-                .getMeetingTemplate();
-        break;
+        case EventRepository.TYPE_MEETING:
+          template = Beans.get(CrmConfigService.class).getCrmConfig(company).getMeetingTemplate();
+          break;
 
-      case EventRepository.TYPE_TASK:
-        template =
-            Beans.get(CrmConfigService.class)
-                .getCrmConfig(event.getUser().getActiveCompany())
-                .getTaskTemplate();
-        break;
+        case EventRepository.TYPE_TASK:
+          template = Beans.get(CrmConfigService.class).getCrmConfig(company).getTaskTemplate();
+          break;
 
-      default:
-        break;
+        default:
+          break;
+      }
+    } else {
+      throw new AxelorException(
+          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
+          I18n.get(BaseExceptionMessage.NO_ACTIVE_COMPANY));
```

Comment:
Please change the message on this method to "Please set an active company for user %s" and set the name of the user set in the event